tests/scripts/setup-pytest-env.sh

```diff
@@ -26,5 +26,20 @@ else
 fi
 set -u
 
+export TVM_TEST_DEVICES=""
+while test $# -gt 0
+do
+    case "$1" in
+        cpu) export TVM_TEST_DEVICES="llvm;llvm -device=arm_cpu;$TVM_TEST_DEVICES"
+            ;;
```

Comment:
I feel it is a bit overly complicated to have two level of directions here:
- D0: arguments in the test script
- D1: env variable(TVM_TEST_DEVICES)
- D2: pytest flag.

It would be awsome if we can reduce it to a single level e.g. an environment variable, and ensure good default behavior so that we can:
- Directly run a specific test via pytest without worrying about setting up the env and run setup-pytest-denv
- Have good default when nothing is set

Reply:
I've removed the need for the pytest flag, its automatically added based on TVM_TEST_DEVICES. We still have to set the environment variable somewhere. Is there a way to do that in the jenkinsfile?

Reply:
The easiest way is to just use another script  e.g. task_integration_gpuonly.sh where we setup the env variable there before source the `task_integration.sh`